doc/classes/NodePath.xml

```diff
@@ -115,72 +130,73 @@
 				[/csharp]
 				[/codeblocks]
 			</description>
 		</method>
 		<method name="get_name_count" qualifiers="const">
 			<return type="int" />
 			<description>
-				Gets the number of node names which make up the path. Subnames (see [method get_subname_count]) are not included.
-				For example, [code]"Path2D/PathFollow2D/Sprite2D"[/code] has 3 names.
+				Returns the number of node names in the path. Property subnames are not included.
+				For example, [code]"../RigidBody2D/Sprite2D:texture"[/code] contains 3 node names.
 			</description>
 		</method>
 		<method name="get_subname" qualifiers="const">
 			<return type="StringName" />
 			<param index="0" name="idx" type="int" />
 			<description>
-				Gets the resource or property name indicated by [param idx] (0 to [method get_subname_count] - 1).
+				Returns the property name indicated by [param idx], starting from 0. If [param idx] is out of bounds, an error is generated. See also [method get_subname_count].
 				[codeblocks]
 				[gdscript]
-				var node_path = NodePath("Path2D/PathFollow2D/Sprite2D:texture:load_path")
-				print(node_path.get_subname(0)) # texture
-				print(node_path.get_subname(1)) # load_path
+				var path_to_name = NodePath("Sprite2D:texture:resource_name")
+				print(path_to_name.get_subname(0)) # Prints "texture"
+				print(path_to_name.get_subname(1)) # Prints "resource_name"
 				[/gdscript]
 				[csharp]
-				var nodePath = new NodePath("Path2D/PathFollow2D/Sprite2D:texture:load_path");
-				GD.Print(nodePath.GetSubname(0)); // texture
-				GD.Print(nodePath.GetSubname(1)); // load_path
+				var pathToName = new NodePath("Sprite2D:texture:resource_name");
+				GD.Print(pathToName.GetSubname(0)); // Prints "texture"
+				GD.Print(pathToName.GetSubname(1)); // Prints "resource_name"
 				[/csharp]
 				[/codeblocks]
 			</description>
 		</method>
 		<method name="get_subname_count" qualifiers="const">
 			<return type="int" />
 			<description>
-				Gets the number of resource or property names ("subnames") in the path. Each subname is listed after a colon character ([code]:[/code]) in the node path.
-				For example, [code]"Path2D/PathFollow2D/Sprite2D:texture:load_path"[/code] has 2 subnames.
+				Returns the number of property names ("subnames") in the path. Each subname in the node path is listed after a colon character ([code]:[/code]).
+				For example, [code]"Level/RigidBody2D/Sprite2D:texture:resource_name"[/code] contains 2 subnames.
 			</description>
 		</method>
 		<method name="hash" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the 32-bit hash value representing the [NodePath]'s contents.
+				Returns the 32-bit hash value representing the node path's contents.
+				[b]Note:[/b] Node paths with equal hash values are [i]not[/i] guaranteed to be the same, as a result of hash collisions. On the contrary, node paths with different hash values are guaranteed to be different.
```

Comment:
```suggestion
				[b]Note:[/b] Node paths with equal hash values are [i]not[/i] guaranteed to be the same, as a result of hash collisions. Node paths with different hash values are guaranteed to be different.
```
While this wording is used elsewhere using "on the contrary" here isn't really correct I'd say, it means "opposite to what was assumed" not "complementary" or "conversely", I think this is sufficient but a different wording could also fit

Reply:
I accept the suggestion. In a future PR, perhaps all of the hash methods could have the wording changed for consistency.

Reply:
Indeed, some of them also have typos with "countrary"